src/noit_check_lmdb.c

```diff
@@ -307,7 +312,8 @@ int noit_check_lmdb_show_checks(mtev_http_rest_closure_t *restc, int npats, char
   return 0;
 }
 
-int noit_check_lmdb_show_check(mtev_http_rest_closure_t *restc, int npats, char **pats) {
+int __noit_check_lmdb_show_check_internal(mtev_http_rest_closure_t *restc, int npats,
```

Comment:
This is not kosher, any symbol in the global scope that starts with `_` is reserved for the compiler and library implementation per ANSI C

If you want to communicate that this is private, please use ``static`` or better yet, move to C++ and put this function as a private method of a class or in a private namespace